cmd/tailscale/cli/up.go

```diff
@@ -647,7 +646,6 @@ func runUp(ctx context.Context, args []string) (retErr error) {
 		if effectiveGOOS() == "windows" {
 			// The Windows service will set this as needed based
```

Comment:
Remove comment (since it was presumably about the clearing of the `StateKey` opt)?